File: InternetBulletin.Shared/DTOs/UserLoginDTO.cs
// *********************************************************************************
//	<copyright file="UserLoginDTO.cs" company="Personal">
//		Copyright (c) 2025 Personal
//	</copyright>
// <summary>User login DTO.</summary>
// *********************************************************************************

namespace InternetBulletin.Shared.DTOs
{
	/// <summary>
	/// User login DTO.
	/// </summary>
	public class UserLoginDTO
	{
		/// <summary>
		/// Gets or sets the user email.
		/// </summary>
		/// <value>
		/// The user email.
		/// </value>
		public string UserEmail { get; set; }

		/// <summary>
		/// Gets or sets the user password.
		/// </summary>
		/// <value>
		/// The user password.
		/// </value>
		public string UserPassword { get; set; }
	}
}

File: IBBS.UI/Models/UserLoginDto.ts
/**
 * UserLoginDtoModel class to represent the user login data.
 */
class UserLoginDtoModel {
    /** The user email. */
    userEmail: string = "";

    /** The user password. */
    userPassword: string = "";

    /**
     * Creates an instance of UserLoginDtoModel
     * @param UserEmail The user email.
     * @param UserPassword The user password.
     */
    constructor(UserEmail: string, UserPassword: string) {
        this.userEmail = UserEmail;
        this.userPassword = UserPassword;
    }
}

export default UserLoginDtoModel;
